### ISSUE TITLE
Can't set camera format, read format is wrong

### ISSUE THREAD

**Issue by jamesej** [l1npengtul/nokhwa#19]
Hi I'm not sure how to make this actionable as I suspect it is a combination issue between the camera on my machine (Surface Laptop, win 10) and the software, here is what I can tell you:

Using Camera::new(0, None), I get
```
CameraFormat w:640 h:480 fmt:MJPEG,
CameraInfo: human_name Microsoft Camera Front desc MediaFoundation Camera Device misc \\?\display#int3470#4&1835d135&0&uid13424#{e5323777-f976-4f5b-9b55-b94699c46e44}\{bf89b5a5-61f7-4127-a279-e187013d7caf} index 0
```
However, Camera.frame_raw() is about 3MB consistently the same size and so not an MJPEG which is confirmed in that
Camera.frame() crashes with
`thread 'test::test_camera' panicked at 'libjpeg fatal error: Not a JPEG file: starts with 0x94 0x94', C:\Users\james\.cargo\registry\src\github.com-1ecc6299db9ec823\mozjpeg-0.8.24\src\errormgr.rs:39:5`

The combination of these issues means I can't get frames in a knowable in advance format, and I don't know how to determine what the format is it give me if it isn't what it tells me when do Camera.camera_format(). Given I get this on the first camera I test i.e. my local machine, and I need this to work for any camera, I'm concluding the library is not usable. Or am I making some stupid mistake?

Do let me know if I can help with any further info.

**Comment by l1npengtul** [comment]
That's odd, it works on my machine with a external Logitech camera. Ill investigate this, thank you!

**Comment by l1npengtul** [comment]
Should be fixed by 7ac349 on 0.9.2 branch, publishing to crates.io soon. Feel free to open another issue if you experience more problems!
